core/templates/pages/exploration-editor-page/services/router.service.ts

```diff
@@ -71,7 +71,9 @@ export class RouterService {
 
     this.location.onPopState(() => {
       if (window.location.hash === '') {
-        window.history.go(-1);
+        setTimeout(() => {
```

Comment:
We generally shouldn't be using a timeout with an arbitrary value. Please could you analyze this and try to understand what the root cause is? Feel free to create a debugging doc if needed.

Reply:
@seanlip can we use setTimeout() without a time value? [Debugging doc](https://docs.google.com/document/d/1Ajqu-AR8UyKHV0ONTntdVkQlRLax7U946DkBqim_kdU/edit?usp=sharing)

Reply:
Yes, that sounds OK. Please add a detailed comment here to explain why you're doing it. Thanks!